mcp/mcp.go

```diff
@@ -7,7 +7,6 @@ import (
 	"errors"
```

Comment:
One minor nit I found in this package: it has `codermcp` as the package name but it's in `mcp`. I think it's better if we name the package `mcp` and alias the other ones we import that conflict.

Reply:
Done!